src/client/pfs_file.go

```diff
@@ -701,9 +701,6 @@ func (c APIClient) DiffFileAll(newCommit *pfs.Commit, newPath string, oldCommit
 
 // WalkFile walks the files under path.
 func (c APIClient) WalkFile(commit *pfs.Commit, path string, cb func(*pfs.FileInfo) error) (retErr error) {
-	defer func() {
-		retErr = grpcutil.ScrubGRPC(retErr)
-	}()
```

Comment:
Why was this removed?  I think we still want to scrub the error here, unless we're going to do a pass across all the client code and make this consistent.

Reply:
This was removed so that the gRPC status code can be inspected.  The test now checks for that.  I don't think we need to move all the error handling over right now.